packages/path_provider/path_provider_android/pubspec.yaml

```diff
@@ -15,11 +15,13 @@ flutter:
       android:
         package: io.flutter.plugins.pathprovider
         pluginClass: PathProviderPlugin
+        dartPluginClass: PathProviderAndroid
 
 dependencies:
   flutter:
     sdk: flutter
-  path_provider_platform_interface: ^2.0.0
+  meta: ^1.3.0
```

Comment:
`meta` shouldn't be necessary if you are just using it for `@visibleForTesting`. You should be able to just import `'package:flutter/foundation.dart'`. I believe the Flutter SDK provides a version of `meta` that we don't want to have version conflicts with later.

Reply:
TIL! We import `meta` in a bunch of plugins, probably mostly for this. We should do a pass to clean that up where possible.

Reply:
Filed https://github.com/flutter/flutter/issues/95658